packages/cli/src/Server.ts

```diff
@@ -554,6 +560,28 @@ class App {
 			});
 		}
 
+		// ----------------------------------------
+		// Public API
+		// ----------------------------------------
+
```

Comment:
wait, is this outdated code? Looks the routes for api-key CRUD are defined in packages/cli/src/UserManagement/routes/users.ts